articles/communication-services/concepts/call-automation/call-automation.md

```diff
@@ -23,14 +23,14 @@ Azure Communication Services(ACS) Call Automation provides developers the abilit
 
 Some of the common use cases that can be built using Call Automation include:
 
-- Program VoIP or PSTN calls for transactional workflows such as click-to-call and appointment reminders to improve customer service.
+- Program VoIP or PSTN calls for transactional workflows, such as click-to-call and appointment reminders, to improve customer service.
 - Build interactive interaction workflows to self-serve customers for use cases like order bookings and updates, using Play (Audio URL) and Recognize (DTMF) actions.
 - Integrate your communication applications with Contact Centers and your private telephony networks using Direct Routing.
-- Protect your customer's identity by building number masking services to connect buyers to sellers or users to partner vendors on your platform.
+- Protect your customers' identity by building number masking services to connect buyers to sellers or users to partner vendors on your platform.
 - Increase engagement by building automated customer outreach programs for marketing and customer service.
-- Analyze in a post-call process your unmixed audio recordings for quality assurance purposes.  
+- Analyze your unmixed audio recordings in a post-call process for quality assurance purposes.  
 
-Azure Communication Services Call Automation can be used to build calling workflows for customer service scenarios, as depicted in the high-level architecture below. You can answer inbound calls or make outbound calls. Execute actions like playing a welcome message, connecting the customer to a live agent on an ACS Calling SDK client app to answer the incoming call request. With support for ACS PSTN or Direct Routing, you can then connect this workflow back to your contact center.  
+Azure Communication Services Call Automation enables the creation of calling workflows for customer service scenarios, as illustrated in the high-level architecture below. It allows for both inbound and outbound calls, as well as the execution of actions such as playing a welcome message and connecting customers to live agents on an Azure Communication Services Calling SDK client app to answer incoming call requests. Additionally, with support for Azure Communication Services PSTN or Direct Routing, you can integrate this workflow with your contact center.  
```

Comment:
Additionally, with support for Azure Communication Services PSTN or Direct Routing, you can integrate this workflow with your contact center to connect with live agents.